### ISSUE TITLE
mini_project_1_He_Li review

### ISSUE THREAD

**Issue by ruxinbu** [helibu/ec601#1]
The code can complete the task successfully. 
The whole structure is very clear, including three different functions and a main function.
It is included a great amount of explanations making reviewers understand the code easily.
The implementation design details are also nice, the code uses Pillow module to draw the text on the pictures, the choice of text size and color are good to see. Also the projection speed is proper.
In conclusion this is a good code for this project.
